uisrnn/utils.py

```diff
@@ -310,3 +300,25 @@ def output_result(model_args, training_args, test_record):
   with open(filename, 'a') as file_object:
     file_object.write(output_string)
   return output_string
+
+
+def estimate_transition_bias(cluster_ids, smooth=1):
+  """estimate transition bias.
+
+  Args:
+    cluster_id: A list of cluster indicator sequences, must be a
+      non-concatenated sequence.
+
```

Comment:
Sorry, I forgot to add docstring. I chose to make the default value 1 because that was the value it had previously in your code (you didn't call it smooth, just added 1). It seemed you put it to avoid -inf value in np.log in the case a sequence with a single speaker is provided. I also thought it was a good idea to add it to the denominator as well to avoid division by 0. Using a smaller value would decrease the bias on the calculation of transition_bias but can also lead to underflow in some very remote cases. I can design some tests that prove my point. 
Do you have any suggestion on the value of `smooth`?

Reply:
I think `smooth=1` is good. Thanks. Please add the docstring to reflect what you explain in this comment.